src/blog/2022/12/community-news-11.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Community News November 2022 #10
+subtitle: News from the FlowForge and Node-RED communities
+description: News from the FlowForge and Node-RED communities
+date: 2022-12-02
+authors: ["rob-marcer"]
+---
+
+Welcome to the FlowForge newsletter for November 2022, a monthly roundup of what’s been happening with both FlowForge and the wider Node-RED community.
+<!--more-->
+If you've got something that you'd like us to share please email [contact@flowforge.com](mailto:contact@flowforge.com).
+
+[**FlowForge 1.1 Released**](http://flowforge.com/blog/2022/10/flowforge-1-1-released/)   
+Version 1.1 was released on 24th November. Our latest release represents our vision of the base set of features needed for you to get great value from using FlowForge in a production environment. That's not to say we are done, we will continue to add features, improve our interfaces and fix bugs with the same enthusiasm as we've worked towards 1.0. We'd like to hear your feedback on what we will be including in [1.1 and beyond](https://github.com/orgs/flowforge/projects/5). 
+
+If you’d like to learn more about what else was included in 1.1 you can do so on our [blog post](http://flowforge.com/blog/2022/10/flowforge-1-1-released), on our [GitHub release page](https://github.com/flowforge/flowforge/releases/tag/v1.1.0) and on our [Youtube channel](https://www.youtube.com/watch?v=134iljE_urI). We’d also love for more of you to get involved in the development of FlowForge, [contributions to the code](https://github.com/flowforge/flowforge/blob/main/CONTRIBUTING.md) and [bug reports](https://github.com/flowforge/flowforge/issues) are really appreciated.
+
+[**Node-RED Nears 3.1 Release**](https://github.com/node-red/node-red/milestone/19)
```

Comment:
Think we should lead with NR 3.1, then FlowForge 1.1, followed by `Node-Redscape` to make it more NR centric. Otherwise LGTM

Reply:
All done, proof read at least 3 times and ready for final sign off hopefully @ZJvandeWeg .